drivers/tee/optee/optee_private.h

```diff
@@ -62,6 +62,7 @@ struct optee_supp {
 
 struct optee {
 	struct tee_device *supp_teedev;
+	bool supp_cnt;
```

Comment:
This should probably go into the `struct optee_supp` above and with `supp_mutex` for synchronization.